lk2nd/panel/generated/lk_panel_otm9605a_lide_qhd_video_550.h

```diff
@@ -0,0 +1,802 @@
+	0x13, 0x00, 0x39, 0xc0, 0xc8, 0x80, 0x98, 0x99,
+	0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x88,
+	0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x67, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_126[] = {
+	0x02, 0x00, 0x39, 0xc0, 0x00, 0x00, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_127[] = {
+	0x02, 0x00, 0x39, 0xc0, 0xc7, 0x00, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_128[] = {
+	0x02, 0x00, 0x39, 0xc0, 0x00, 0xb1, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_129[] = {
+	0x02, 0x00, 0x39, 0xc0, 0xc6, 0x06, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_130[] = {
+	0x02, 0x00, 0x39, 0xc0, 0x00, 0xb4, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_131[] = {
+	0x02, 0x00, 0x39, 0xc0, 0xc6, 0x10, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_132[] = {
+	0x02, 0x00, 0x39, 0xc0, 0x00, 0x00, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_133[] = {
+	0x22, 0x00, 0x39, 0xc0, 0xec, 0x40, 0x34, 0x44,
+	0x44, 0x43, 0x44, 0x34, 0x44, 0x44, 0x43, 0x44,
+	0x34, 0x44, 0x44, 0x43, 0x44, 0x34, 0x44, 0x34,
+	0x44, 0x44, 0x43, 0x44, 0x34, 0x44, 0x44, 0x43,
+	0x44, 0x34, 0x44, 0x44, 0x43, 0x04, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_134[] = {
+	0x02, 0x00, 0x39, 0xc0, 0x00, 0x00, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_135[] = {
+	0x22, 0x00, 0x39, 0xc0, 0xed, 0x40, 0x34, 0x44,
+	0x34, 0x44, 0x34, 0x44, 0x34, 0x44, 0x44, 0x43,
+	0x44, 0x43, 0x44, 0x43, 0x44, 0x34, 0x44, 0x34,
+	0x44, 0x34, 0x44, 0x34, 0x44, 0x44, 0x43, 0x44,
+	0x43, 0x44, 0x43, 0x44, 0x43, 0x04, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_136[] = {
+	0x02, 0x00, 0x39, 0xc0, 0x00, 0x00, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_137[] = {
+	0x22, 0x00, 0x39, 0xc0, 0xee, 0x40, 0x44, 0x44,
+	0x44, 0x44, 0x44, 0x44, 0x44, 0x44, 0x44, 0x44,
+	0x44, 0x44, 0x44, 0x44, 0x44, 0x44, 0x44, 0x44,
+	0x44, 0x44, 0x44, 0x44, 0x44, 0x44, 0x44, 0x44,
+	0x44, 0x44, 0x44, 0x44, 0x44, 0x04, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_138[] = {
+	0x02, 0x00, 0x39, 0xc0, 0x00, 0x00, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_139[] = {
+	0x04, 0x00, 0x39, 0xc0, 0xff, 0xff, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_140[] = {
+	0x02, 0x00, 0x39, 0xc0, 0x51, 0x00, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_141[] = {
+	0x02, 0x00, 0x39, 0xc0, 0x53, 0x24, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_142[] = {
+	0x02, 0x00, 0x39, 0xc0, 0x55, 0x01, 0xff, 0xff
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_143[] = {
+	0x11, 0x00, 0x05, 0x80
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_144[] = {
+	0x29, 0x00, 0x05, 0x80
+};
+static char otm9605a_lide_qhd_video_550_on_cmd_145[] = {
+	0x51, 0xfa, 0x15, 0x80 /* https://github.com/msm8916-mainline/lk2nd/commit/3d8b68ddef17afeaba7b34180489370598cdaaea */
```

Comment:
```suggestion
	0x51, 0xff, 0x15, 0x80 /* https://github.com/msm8916-mainline/lk2nd/commit/3d8b68ddef17afeaba7b34180489370598cdaaea */
```

The maximum is 255. See `otm9605a_lide_qhd_video_550_backlight`